source/module_base/inverse_matrix.h

```diff
@@ -20,12 +20,12 @@ class Inverse_Matrix_Complex
 	void init( const int &dim_in);
 
 	private:
-	int dim;
-	double *e;
-	int lwork;
-	std::complex<double> *work2;
-	double* rwork;
-	int info;
+	int dim{};
+	double *e{};
+	int lwork{};
+	std::complex<double> *work2{};
+	double* rwork{};
```

Comment:
why not use nullptr here?

Reply:
Using `{}` is identical, as it [initializes the pointer to nullptr](https://en.cppreference.com/w/cpp/language/zero_initialization#:~:text=A%20zero%2Dinitialized%20pointer%20is%20the%20null%20pointer%20value%20of%20its%20type%2C%20even%20if%20the%20value%20of%20the%20null%20pointer%20is%20not%20integral%20zero.).

Reply:
I am not looking identical ways of writing C++ codes, I just consider this new way of initializing variables make reading more difficult for some people, we should have a unified way. For example, double *rwork = nullptr;